_includes/meshery.html

```diff
@@ -292,7 +292,7 @@ <h4>Brew user</h4>
 
         <div id="start-brew">
             <div style="width:auto; height:auto; padding:0px; position:relative;">
-                <pre><code style="color:#eee;"></br> $ brew install layer5io/mesheryctl</br> $ mesheryctl system start
+                <pre><code style="color:#eee;"></br> $ brew tap layer5io/tap</br> $ brew install mesheryctl</br> $ mesheryctl system start
           </code></pre>
                 <!-- copy to clipboard -->
                 <a class="btn clipboard-copy-btn tooltip" style="background:none; border: 0;" data-clipboard-target="#start-brew" data-clipboard-text="brew install layer5io/mesheryctl
```

Comment:
The `data-clipboard-text` attribute in this line also needs to be updated.

Reply:
Ah yes, forgot thank, thank you!